### ISSUE TITLE
Math Operations

### ISSUE THREAD

**Issue by Krafty-Coder** [Abdur-rahmaanJ/greenBerry#20]
Add functionality code for math operations.

**Comment by KushalVijay** [comment]
Hello!!, I am a beginner to open source and pretty good at python. What can I do to contribute  or where should i code to resolve this issue.

Regards 
Kushal Vijay

**Comment by Abdur-rahmaanJ** [comment]
also see update  README

**Comment by KushalVijay** [comment]
Okay. I will go through it and try to work on it.

**Comment by Josephasafg** [comment]
is this already taken by @KushalVijay ?

**Comment by KushalVijay** [comment]
No, the issue given to me has already been resolved. You can take over it @Josephasafg

**Comment by godspeedfw** [comment]
Is this still an issue? If so I would like to work on this.
Thank you

**Comment by Abdur-rahmaanJ** [comment]
@0xGodspeed Try this one:

https://github.com/Abdur-rahmaanJ/greenberry/issues/80
